code/game/atoms_movable.dm

```diff
@@ -126,7 +126,7 @@
 		destination.Entered(src, oldloc)
 		var/area/old_area = get_area(oldloc)
 		var/area/destarea = get_area(destination)
-		if(old_area != destarea)
+		if(destarea && old_area != destarea)

```

Comment:
Yes. Sort of? This might only happen on Runtime station, but firing limbs runtimes there, and I figured it must happen for other things.